sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -244,17 +271,26 @@ const formatFormData = (data: FormListing, units: TempUnit[]) => {
     delete unit.tempId
   })
 
+  const events: ListingEventCreate[] = []
+  if (data.lotteryDate && data.reviewOrderQuestion === "reviewOrderLottery") {
+    const startTime = createTime(createDate(data.lotteryDate), data.lotteryStartTime)
+    const endTime = createTime(createDate(data.lotteryDate), data.lotteryEndTime)
+
+    events.push({
+      type: ListingEventType.publicLottery,
+      startTime: startTime,
+      endTime: endTime,
+      note: data.lotteryDateNotes,
+    })
+  }
```

Comment:
If there are other events on the listing, won't this wipe them out?

Reply:
to my knowledge we aren't saving any other events right now, unless that was added in a different PR after i rebased

Reply:
Ouch, this code just bit me in #1568…I'll write more about it there.